src/matrixrtc/CallMembership.ts

```diff
@@ -14,35 +14,69 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { EitherAnd } from "matrix-events-sdk/lib/types";
-
 import { MatrixEvent } from "../matrix.ts";
 import { deepCompare } from "../utils.ts";
 import { Focus } from "./focus.ts";
 import { isLivekitFocusActive } from "./LivekitFocus.ts";
 
-type CallScope = "m.room" | "m.user";
-// Represents an entry in the memberships section of an m.call.member event as it is on the wire
-
-// There are two different data interfaces. One for the Legacy types and one compliant with MSC4143
+export const DEFAULT_EXPIRE_DURATION = 1000 * 60 * 60 * 4; // 4 hours
```

Comment:
As an exported member, could this some docs saying what it is other than it being 4 hours?

Reply:
This export got removed entirely when resolving https://github.com/matrix-org/matrix-js-sdk/pull/4583#discussion_r1904376447.

When it gets added back in we will make sure that it is properly documented 👍